src/utils/file-upload-utils.js

```diff
@@ -1,3 +1,7 @@
+import logger from "@logger/logger"
+
+const { CM_API_KEY } = process.env
```

Comment:
Curious, what happens if this key does not exist. just thinking if it will break the program. do we have a "env" checker function when our server starts up.

That will give us confidence we can directly unwrap the env vars